lib/server-api.d.ts

```diff
@@ -256,9 +256,6 @@ declare module Server{
 		importProvision(provision: any): IFuture<Server.ProvisionData>;
 		removeProvision(identifier: string): IFuture<void>;
 	}
-	interface INativescriptServiceContract{

```

Comment:
Do you need to remove this? It seems that the swagger api is generated based on branch from Ice repo that is not rebased on release.